### PR TITLE
Feature/multi lang template

### PR DESCRIPTION
**Pull request by b1ff**

### CODE REVIEW

File: registry/server/migrations/20220323123155_add-locale-to-template.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Knex } from "knex";
+import { tables } from '../db/structure';
+
+
+export async function up(knex: Knex): Promise<void> {
+    return knex.schema.createTable(tables.templatesLocalized, table => {
+        table.string('templateName', 50).notNullable().references('templates.name').onDelete('CASCADE');
+        table.text('content').notNullable();
```
Comment (blackrabbit99) [inline registry/server/migrations/20220323123155_add-locale-to-template.ts:8]: `table.text('content', 'mediumtext').notNullable();`

Comment (b1ff) [reply registry/server/migrations/20220323123155_add-locale-to-template.ts:8]: what means 'medumtext'?

Comment (b1ff) [reply registry/server/migrations/20220323123155_add-locale-to-template.ts]: found answer here https://github.com/knex/knex/issues/1064#issuecomment-797181826 
Updated type

**Review by blackrabbit99 — APPROVED** [review]

File: ilc/server/errorHandler/ErrorHandler.spec.js [hunk 2]
```diff
@@ -43,6 +43,7 @@ describe('ErrorHandler', () => {
         });
 
         const response = await server.get('/_ilc/500').expect(500);
+        console.log(response.text);
```
Comment (yehor-manzhula) [inline ilc/server/errorHandler/ErrorHandler.spec.js:46]: Is it expected to be here?

File: registry/server/templates/interfaces/index.ts [hunk 3]
```diff
@@ -3,10 +3,16 @@ import {getJoiErr} from '../../util/helpers';
...
-    name: string,
-    content: string,
+    name: string;
+    content: string;
 };
 
+export interface LocalizedTemplate {
+    templateName: string;
+    content: string;
+    locale?: string;
```
[the remark this answers is not in the document]
Comment (b1ff) [reply registry/server/templates/interfaces/index.ts:13]: default locale is specified on `content` column of the `template` table

[the remark this answers is not in the document]
Comment (b1ff) [reply registry/server/templates/interfaces/index.ts:13]: But you are right it should be required

File: ilc/server/errorHandler/ErrorHandler.spec.js (hunk 2 above)
Comment (b1ff) [reply ilc/server/errorHandler/ErrorHandler.spec.js]: nope

Comment (b1ff) [reply ilc/server/errorHandler/ErrorHandler.spec.js]: fixes

File: registry/server/templates/routes/createTemplate.ts [hunk 4]
```diff
@@ -6,24 +6,47 @@ import {
...
+import { readTemplateWithAllVersions } from '../services/templatesRepository';
 
 const validateRequestBeforeCreateTemplate = validateRequestFactory([{
     schema: templateSchema,
     selector: 'body',
 }]);
 
 const createTemplate = async (req: Request, res: Response): Promise<void> => {
-    const template = req.body;
+    const request = req.body;
```
[the remark this answers is not in the document]
Comment (b1ff) [reply registry/server/templates/routes/createTemplate.ts:21]: didn't get this one

**Review by yehor-manzhula — APPROVED** [review]

**Pull request merged**